VTEX - Antifraud Provider API.json

```diff
@@ -1472,19 +1434,18 @@
                     "returnUrl"
                 ],
                 "type": "object",
+                "description": "Token information.",
                 "properties": {
                     "applicationId": {
                         "type": "string",
-                        "description": "This value is always identified as `vtex`."
+                        "description": "This value is always identified as `vtex`.",
+                        "example": "vtex"
                     },
                     "returnUrl": {
                         "type": "string",
-                        "description": "VTEX website URL. You will redirect the user after they complete login on the payment provider's website. The URL should contain a query string parameter called `authorizationCode` which will be passed empty and which you must fill in before returning the user."
+                        "description": "VTEX website URL. You will redirect the user after they complete login on the payment provider's website. The URL should contain a query string parameter called `authorizationCode` which will be passed empty and which you must fill in before returning the user.",
```

Comment:
Acredito que no final deveria ser "returning **to** the user".

Reply:
Corrigido.